packages/devtools_server/lib/src/client_manager.dart

```diff
@@ -90,6 +92,17 @@ class DevToolsClient {
           _vmServiceUri = null;
           _respond(request);
           return;
+        case 'getPreferenceValue':
+          final String key = _currentPage = request['params']['key'];
+          final dynamic value = ServerApi.devToolsPreferences.properties[key];
+          _respondWithResult(request, value);
+          return;
+        case 'setPreferenceValue':
+          final String key = _currentPage = request['params']['key'];
+          final dynamic value = _currentPage = request['params']['value'];
```

Comment:
Are these `_currentPage =` supposed to be here? It looks like this will overwrite the `_currentPage` variable with the keys/values being stored?

Reply:
thanks for the eyes on this; this wasn't correct (and I'm not even sure how I wrote this; copy pasta? a merge issue?)